README.md

```diff
@@ -51,13 +51,9 @@ Detailed documentation on the Import Tool is available [here](docs/usage.md).
    - For MacOS: `brew install maven`
 2. The tools use protobuf and require that `protoc` be installed.
    - For MacOS: `brew install protobuf`
-3. Make sure Java 11+ (but not Java 16) is installed
+3. Make sure Java 11+ is installed
```

Comment:
shall we suggest always use 16+?

Reply:
We probably should support a wide range of Java versions... Maybe leave it for now...